support/doc/production.md

```diff
@@ -70,6 +70,8 @@ configuration.
 
 ### Webserver
 
+We maintain a few example configurations in `support/doc/`, for different web servers. Here we depict the installaton procedure with the one of Nginx.
```

Comment:
We only support nginx (and we don't want to support other webservers) so I don't think this is useful :)

Reply:
Didn't know that :)